tool/tbot/main.go

```diff
@@ -144,6 +155,10 @@ func onWatch(botConfig *config.BotConfig) error {
 }
 
 func onStart(botConfig *config.BotConfig) error {
+	if botConfig.AuthServer == "" {
+		return trace.BadParameter("An auth or proxy server must be set via --auth-server or configuration")
```

Comment:
nit: uppercase error string

```suggestion
		return trace.BadParameter("an auth or proxy server must be set via --auth-server or configuration")
```